<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsTechnicalInfo</class>
 <widget class="QWidget" name="SettingsTechnicalInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="settingsSectionHeader" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="SilentQPushButton" name="backBtn">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Atrás</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sectionTitleLbl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10000</height>
         </size>
        </property>
        <property name="text">
         <string>Información básica</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="settingsMarginTop">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="settingsMenuListCont" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="idDeviceLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>ID dispositivo:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="idDeviceValLbl">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>bqOS...</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="softwareLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Software:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="softwareValLbl">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="MACadressLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Direción MAC:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MACaddressValLbl">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="internalMemLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Memoria interna:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="internalMemValLbl">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="externalMemLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Memoria Externa:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="externalMemValLbl">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="internalVersionLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>198</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Internal version:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="internalVersionValLbl">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="rootFsLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Rootfs version:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rootFsValLbl">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="kernelVersionLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Kernel version:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="kernelVersionVaLbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetCont" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="bateryLbl">
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>228</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Nivel de batería:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="batteryValLbl">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>100%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalBottomSection">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SilentQPushButton</class>
   <extends>QPushButton</extends>
   <header>SilentQPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
